src/num/theories/cv_compute/automation/Holmakefile

```diff
@@ -0,0 +1,22 @@
+all: $(DEFAULT_TARGETS) selftest.exe
+
+.PHONY: all link-to-sigobj
+
+ifdef HOLBUILD
+all: link-to-sigobj
+.PHONY: link-to-sigobj
+link-to-sigobj: $(DEFAULT_TARGETS)
+	$(HOL_LNSIGOBJ)
+endif
+
+selftest.exe: selftest.uo
```

Comment:
@mn200 should this line have more deps?

Reply:
As long as selftest.sml clearly depends (via on `open fooTheory` type line) on what's needed, this should be fine.